themes/aframe/layout/partials/blog/twia_projects.ejs

```diff
@@ -0,0 +1,10 @@
+<% if (projects) { %>
+  <h3>Projects</h3>
+  <ul>
+    <% projects.forEach(function (project) { %>
+      <li>
+        <%- project.author %> built <a href="<%- project.url %>"><%- project.title %></a> - <%- project.description %>
```

Comment:
don't we want to link to Author's pages?

![image](https://cloud.githubusercontent.com/assets/203725/14442036/8be43590-ffec-11e5-8d2a-2c9a79ae95f8.png)